spec.emu

```diff
@@ -102,97 +112,166 @@ contributors: Google, Ecma International
             </tr>
           </thead>
           <tr>
-            <td>`"sunday"`</td>
-            <td>Name indicating Sunday, e.g. Su, Sun, Sunday, etc.</td>
-          </tr>
-          <tr>
-            <td>`"monday"`</td>
+            <td>`"1"`</td>
```

Comment:
Should be a number, not a string, right?  Or is the argument coerced to a string?

Reply:
ToString is called in Intl.DisplayNames.prototype.of( code ) first then call into IsValidCodeForDisplayNames. Therefore, all the code is already turned into String when it passing to IsValidCodeForDisplayNames, even if the input code user passed in is a number.